Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.py

```diff
@@ -804,9 +804,10 @@ def get_modified_incidents_close_or_repopen_entries(modified_incidents: List[dic
     entries = []
     if close_incident:
         for incident in modified_incidents:
-            if incident.get("status") == "Resolved":
+            if incident.get("status") in ("Resolved", "Redirected"):
                 demisto.debug(
-                    f"Microsoft Defender 365 - incident {incident.get(MICROSOFT_INCIDENT_ID_KEY)} is resolved in Microsoft, "
+                    f"Microsoft Defender 365 - incident {incident.get(MICROSOFT_INCIDENT_ID_KEY)} is "
+                    f"{incident.get('status').lower()} in Microsoft, "
```

Comment:
```suggestion
                    f"{(incident.get('status') or "").lower()} in Microsoft, "
```
This will solve the pre commit issue